core/tests/test_utils.py

```diff
@@ -1880,7 +1881,7 @@ def save_new_topic_with_subtopic_schema_v1(
 
     def save_new_question(
             self, question_id, owner_id, question_state_data,
-            linked_skill_ids,
+            linked_skill_ids, inapplicable_misconception_ids,
```

Comment:
Could we use some default value for `inapplicable_misconception_ids`? I think the benefit of these test_utils functions is that you only need to specify a limited number of things and the rest is set to some default values.

Reply:
This is weird.. I was not able to use a list as a default param. There was a lint error.

Reply:
![image](https://user-images.githubusercontent.com/11008603/91656340-ac6d4f80-ead5-11ea-842d-f0f7e980bd3b.png)

This is what I meant.

Reply:
I was not able to find a workaround for this case.

Reply:
Yes, do not use a list as a default param. It is indeed dangerous. See https://docs.python-guide.org/writing/gotchas/.

I think it is fine to pass in manually.

Reply:
Yes, I know. Thanks. I have spoken to @vojtechjelinek about this offline and he recommends passing `None` and reassigning to [] inside the method.

Reply:
Done.